src/i18n/messages/en.messages.json

```diff
@@ -13,14 +13,14 @@
       "title": "History Queries"
     },
     "nextQueries": {
-      "title": "Next Queries"
+      "title": "What's Next"
     },
     "nextQueriesGroup": {
       "title": "You may be interested",
-      "message": "This is what other shoppers searched after <span class=\"x-text x-text--bold\">\"{query}\"</span>"
+      "message": "Those who searched for <span class=\"x-text x-text--bold\">\"{query}\"</span> also viewed:"
     },
     "identifierResults": {
```

Comment:
Remove the idenftifierResults translation

Reply:
done